doc/classes/EditorExportPlugin.xml

```diff
@@ -34,8 +34,9 @@
 			<param index="0" name="resource" type="Resource" />
 			<param index="1" name="path" type="String" />
 			<description>
-				Customize a resource. If changes are made to it, return the same or a new resource. Otherwise, return [code]null[/code].
-				The [i]path[/i] argument is only used when customizing an actual file, otherwise this means that this resource is part of another one and it will be empty.
+				Customize a resource. If changes are made to it, return the same or a new resource. Otherwise, return [code]null[/code]. When a new resource is returned, [param resource] will be replaced by a copy of the new resource.
+				When customizing a [CompressedTexture2D] and returning a new resource, [method skip] should not be called on this new resource from [method _export_file] because [param resource] will now point to its [code].ctex[/code] file. In this scenario, the [code].ctex[/code] file that was initially referenced by [param resource] will no longer be included in the project.
```

Comment:
```suggestion
				[b]Note:[/b] When customizing a [CompressedTexture2D] and returning another resource, the other resource should not be skipped using [method skip] in [method _export_file].
```
The extra details are unnecessary IMO.

Does this also apply when the other texture is not a CompressedTexture2D? Are there any other Resource types with this caveat?

Reply:
> The extra details are unnecessary IMO.

Yeah, I think you’re right: your suggested text heavily the rest without need to explicitly write it.

> Does this also apply when the other texture is not a CompressedTexture2D? Are there any other Resource types with this caveat?

Good question. I can do some trial and error with all the file types supported by Godot to see how they behave; I think I’ll need to do this at some point anyway to be sure my resource remaps plugin doesn’t need any additional special logic.

Reply:
I found there are a few more resource types that can’t be skipped if they have been customized. I’ll push an update to include these types on Monday when I’m back in the office.